AdblockPlusSafari/AdblockPlus.m

```diff
@@ -26,6 +26,8 @@
 static NSString *AdblockPlusDownloadedVersion = @"AdblockPlusDownloadedVersion";
 static NSString *AdblockPlusWhitelistedWebsites = @"AdblockPlusWhitelistedWebsites";
 
+static NSString *AdblockPlusSafariExtension = @"AdblockPlusSafariExtension";
```

Comment:
Sorry, missed this comment because GitHub didn't send out a notification. Yeah, if it makes sense let's keep it.